kairon/shared/pos/processor.py

```
@@ -866,5 +879,118 @@ def reject_pos_order(self, session_id: str, order_id: int) -> Dict[str, Any]:
+        if not data:
+            raise Exception(f"Group XML-ID '{xml_id}' not found")
+
+        return data[0]["res_id"]
+
+    def create_user(
+            self,
+            session_id: str,
+            login: str,
+            password: str,
+            name: str,
+            partner_id: int = None,
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)